guides/source/asset_pipeline.md

```diff
@@ -1101,7 +1101,7 @@ Windows you have a JavaScript runtime installed in your operating system.
 
 
 
-### Serving GZipped version of assets
+### Generating GZipped version of assets
```

Comment:
I think calling it `GZipping your assets` might be best, and then adding some text below the code block addressing how to actually serve gzipped assets. Something like `Refer to your web server's documentation for instructions on how to serve GZipped assets.` maybe?

Reply:
Yes, this sounds good to me. I will make a change and push it today :)